webhook-service/src/main/scala/io/renku/webhookservice/hookdeletion/ProjectHookDeletor.scala

```diff
@@ -61,17 +57,15 @@ private class ProjectHookDeletorImpl[F[_]: Async: Logger](
   }
 
   def delete(projectId: projects.Id, projectHookId: HookIdAndUrl, accessToken: AccessToken): F[DeletionResult] =
-    for {
-      uri    <- validateUri(s"$gitLabUrl/api/v4/projects/$projectId/hooks/${projectHookId.id}")
-      result <- send(request(DELETE, uri, accessToken))(mapResponse)
-    } yield result
+    gitLabClient.delete(uri"projects" / projectId.show / "hooks" / projectHookId.id.show, "project hooks")(mapResponse)(
```

Comment:
Perhaps, the name of the GitLab API should be distinguishing the hook deletion from creation. I think now we use the same name for both calls.